src/test/resources/net/sf/jabref/logic/exporter/BibTeXMLExporterTestInCollection.bib

```diff
@@ -0,0 +1,16 @@
+% Encoding: UTF-8
+
+@InCollection{Mustermann2016,
+  author =    {Max Mustermann},
+  title =     {Java InCollection},
+  booktitle = {Java Testing},
+  publisher = {Java Publisher},
+  year =      {2016},
+  editor =    {Maxima Musterfrau},
+  volume =    {1},
+  number =    {1},
+  chapter =   {3},
```

Comment:
Yeah, this is intentionally, because the schema doesn't have something like chapter in the InCollection type.